### PR TITLE
Fixing photometry plot issues

### PR DESCRIPTION
**Pull request by Will-Cooper**
Dealing with #108, CAMD plots having issues on solo results.
Minor security update to requirements

### CODE REVIEW

File: simple_app/utils.py [hunk 1]
```diff
@@ -491,32 +491,32 @@ def absmags(df: pd.DataFrame, all_bands: np.ndarray) -> pd.DataFrame:
         The output dataframe with absolute mags calculated
     """
 
-    def pogsonlaw(m: Union[float, np.ndarray], dist: Union[float, np.ndarray]) -> Union[float, np.ndarray]:
+    def pogsonlaw(m: Union[float, np.ndarray]) -> Union[float, np.ndarray]:
         """
         Distance modulus equation
```
Comment (kelle) [inline simple_app/utils.py:496]: Could you comment this more. It's more than just the distance modulus equation. It uses the parallax to calculate a distance. But returns a Nan if the distance is 0 or negative?

File: simple_app/utils.py [hunk 2]
```diff
@@ -491,32 +491,32 @@ def absmags(df: pd.DataFrame, all_bands: np.ndarray) -> pd.DataFrame:
...
         -------
         _
             Absolute magnitude
         """
-        return np.where(dist > 0, m - 5 * np.log10(dist, where=dist > 0) + 5, np.nan)
+        return np.where(df.parallax > 0, m + 5 * np.log10(df.parallax, where=df.parallax > 0) - 10, np.nan)
 
-    df['dist'] = np.divide(1000, df.parallax, where=df.parallax > 0)
     dmags: Dict[str, np.ndarray] = {}
     for mag in all_bands:
```
Comment (kelle) [inline simple_app/utils.py:510]:
```suggestion
    for band in all_bands:
```

File: simple_app/utils.py [hunk 3]
```diff
@@ -491,32 +491,32 @@ def absmags(df: pd.DataFrame, all_bands: np.ndarray) -> pd.DataFrame:
...
         abs_mag = "M_" + mag
```
Comment (kelle) [inline simple_app/utils.py:511]:
```suggestion
        abs_mag = "M_" + band
```

File: simple_app/utils.py [hunk 4]
```diff
@@ -491,32 +491,32 @@ def absmags(df: pd.DataFrame, all_bands: np.ndarray) -> pd.DataFrame:
...
-            dmags[abs_mag] = pogsonlaw(df[mag], df['dist'])
+            dmags[abs_mag] = pogsonlaw(df[mag])
         except KeyError:
-            dmags[abs_mag] = pogsonlaw(df[mag[:mag.find('(')]], df['dist'])
-    df = pd.concat([df, pd.DataFrame.from_dict(dmags)], axis=1)
+            dmags[abs_mag] = pogsonlaw(df[mag[:mag.find('(')]])
+    dfabs = pd.DataFrame.from_dict(dmags)
+    if 'magnitude' in df.index:
+        dfabs.rename(index={0: 'magnitude'}, inplace=True)
+    df = pd.concat([df, dfabs], axis=1)
```
Comment (kelle) [inline simple_app/utils.py:519]: Could you add some more comments here. I think it might be helpful if the variable names were more descriptive.

**Review by kelle — APPROVED** [review]
I left some rather nitpicky comments which I don't think impact the functionality. If you have time to address, GREAT. if not, this is ok to merge. We can polish the code another time.

**Comment by dr-rodriguez** [comment]
I didn't get a chance to look at this in detail (reading the code is a bit of a challenge; see #128 for a ticket to help that), but I did notice you updated astrodbkit2 to 0.5 which uses SQLAlchemy 2.0. I think that's OK, but just a heads up if you see any error messages it could be since SQLAlchemy 2.0 has changed how some DB calls work (generally for inserts/updates so you should be fine).

**Pull request merged**